stories/Content/Tag/SimpleTag/SimpleTag.stories.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { SimpleTag as Component } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { SimpleTag as Component } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Simple Tag",
+  title: "Content/Tag/Simple Tag/Simple Tag",
```

Comment:
```suggestion
  title: "Content/Tag/Simple Tag",
```

Simple Tag stories from Tag QA should use this title too.

Reply:
fixed